pages/comissao.vue

```diff
@@ -61,38 +61,7 @@
             </div>
           </v-card-text>
         </v-card>
-        <v-card-actions>
-          <v-container>
-            <v-icon
-              class="mx-5"
-              tag="a"
-              href="https://www.facebook.com/recepcaoimeusp"
-              target="_blank"
-              color="#626262"
-              size="35px"
-              >fa-brands fa-facebook</v-icon
-            >
-
-            <v-icon
-              class="mx-5"
-              tag="a"
-              href="https://www.instagram.com/imeusp.recepcao/"
-              target="_blank"
-              color="#626262"
-              size="35px"
-              >fa-brands fa-instagram</v-icon
-            >
-            <v-icon
-              class="mx-5"
-              tag="a"
-              href="https://discord.gg/qKXAkf8EjE"
-              target="_blank"
-              color="#626262"
-              size="35px"
-              >fa-brands fa-discord</v-icon
-            >
-          </v-container>
-        </v-card-actions>
+        <SocialMedia> </SocialMedia>
```

Comment:
Como não temos nada "dentro" do componente (não há [slots](https://br.vuejs.org/v2/guide/components-slots.html)), podemos fechá-lo na mesma tag.

```suggestion
        <SocialMedia />
```